packages/server/routes/test/button.tsx

```
@@ -0,0 +1,21 @@
+
+const Button = ({ name }: { name: string }) => {
+  const [count, setCount] = useState(0);
+
+  const handleClick = () => {
+    window.alert("Button:" + name + " clicked!");
+    setCount(count + 1);
+    console.log("Count:", count);
+  };
+
+  return (
+    <button onClick={handleClick}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)